build.gradle

```
@@ -9,14 +9,29 @@ plugins {
   id 'org.sonarqube' version '3.3'
   id 'uk.gov.hmcts.java' version '0.12.2'
   id 'com.github.spacialcircumstances.gradle-cucumber-reporting' version '0.1.23'
+  id 'au.com.dius.pact' version '4.1.0'
 }
 
+ext['springCloudVersion'] = '2020.0.1'
+ext['springsecurityVersion'] = '5.5.1'
+ext['swagger2Version'] = '3.0.0'
+ext['diuspactproviderVersion'] = '4.2.14'
+
 group = 'uk.gov.hmcts.reform'
 version = '0.0.1'
 
 sourceCompatibility = 11
 targetCompatibility = 11
 
+configurations {
+  providedRuntime
+  annotationProcessor
+
+  contractTestCompile.extendsFrom(testCompile)
+  contractTestRuntime.extendsFrom(testRuntime)
+  cucumberRuntime.extendsFrom(functionalRuntime)
```

```suggestion
  contractTestCompile.extendsFrom(testCompile)
  contractTestRuntime.extendsFrom(testRuntime)
```
The rest don't seem to be related to PACT